hail/python/hail/docs/install/macosx.rst

```diff
@@ -9,8 +9,7 @@ Install Hail on Mac OS X
 
   .. code-block::
 
-    brew tap homebrew/cask-versions
-    brew install --cask temurin8
+    brew install temurin@11
```

Comment:
We should make this consistent with the recommendation above, which currently suggests Azul. I don't have a strong preference, but I've been using Azul myself for several years.

Reply:
Actually, I believe Azul is what we use in our production images, so that's a reason to default to it locally as well.

Reply:
After some more digging, we do use openjdk in our production images, and recommended Azul here because at the time openjdk didn't offer m1 native builds. That's no longer true, so I'd default back to providing a link to a Temurin package above: https://adoptium.net/temurin/releases/?os=mac&arch=aarch64&version=11

Reply:
Or I think Azul is fine too, as that's what we all use for development. 🤷